packs/bit-bucket/actions/list_issues.py

```diff
@@ -0,0 +1,19 @@
+from lib.action import BitBucketAction
+
+
+class ListIssuesAction(BitBucketAction):
+    def run(self, repo):
+        """
+        List Issues of Repository with title
+        of the issue its status and reporter
+        """
+        bb = self.perform_request(repo=repo)
+        success, result = bb.issue.all()
+        issues = {}
+        for i in range(result['count']):
+            issues[result['issues'][i]['title']] = {
```

Comment:
Also, similar as above, we could simple return the whole issue object as returned by the API instead of only picking a couple of keys / attributes from it.